forge-gui/res/tokenscripts/role_questing.txt

```diff
@@ -0,0 +1,12 @@
+Name:Questing
+ManaCost:no cost
+Types:Enchantment Aura Role
+K:Enchant creature
+A:SP$ Attach | Cost$ 0 | ValidTgts$ Creature | AILogic$ Pump
+S:Mode$ Continuous | Affected$ Creature.EnchantedBy | AddKeyword$ Vigilance & Deathtouch & Haste | Description$ Enchanted creature has vigilance, deathtouch, and haste.
+S:Mode$ CantBlockBy | ValidAttacker$ Creature.EnchantedBy | ValidBlocker$ Creature.powerLE2 | Description$ Enchanted creature can't be blocked by creatures with power 2 or less.
+S:Mode$ CantPreventDamage | IsCombat$ True | ValidSource$ Creature.YouCtrl | Description$ Combat damage that would be dealt by creatures you control can't be prevented.
+T:Mode$ DamageDone | ValidSource$ Creature.EnchantedBy | ValidTarget$ Opponent | CombatDamage$ True | TriggerZones$ Battlefield | Execute$ MoreDamage | TriggerDescription$ Whenever enchanted creature deals combat damage to an opponent, it deals that much damage to target planeswalker that player controls.
+SVar:MoreDamage:DB$ DealDamage | ValidTgts$ Planeswalker.ControlledBy TriggeredTarget | TgtPrompt$ Select target planeswalker that player controls | NumDmg$ X
```

Comment:
Don't know if the Trigger should also be gained to the Enchanted Creature?

Or if the DealDamage needs extra Damage Source

Reply:
Oh yeah you're right, it should be giving the abilities to the creature it'd make more sense given the wording.
Was kinda short-circuiting on those las vegas cards.